Component/SchemaFilter.php

```diff
@@ -86,18 +86,20 @@ public function processSchemaBaseConfig(array $schemaConfig, $schemaName)
         if (!empty($schemaConfig['tableFields'])) {
             $schemaConfig['table'] += array('columns' => $schemaConfig['tableFields']);
         }
-        foreach ($schemaConfig['table']['columns'] as $k => $column) {
-            // Resolve mapping-style vs list style data attribute specification
-            if (empty($column['data'])) {
-                $schemaConfig['table']['columns'][$k]['data'] = $k;
+        if ($schemaConfig['table']['columns'] ?? false) {
```

Comment:
this will yield an error if table or columns is not defined. Maybe replace with `isset($schemaConfig['table']['columns'])`?

Reply:
should have exactly the same effect as `isset($schemaConfig['table']['columns'])` (view https://www.decodingweb.dev/php-double-question-marks)

Reply:
nice, this is very convenient, I thought the `??`operator behaves like the `?:`. Thanks for pointing out!